static/js/src/elements/TextInput.vue

```diff
@@ -50,6 +51,12 @@ export default {
       type: String,
       default: 'text',
     },
+
+    inputmode: {
+      type: String,
+      default: null,
```

Comment:
I typically use `false` when I want an attribute not to appear. But it doesn't matter much.

Reply:
I saw that vue docs said to be empty it could be `null`, `undefined`, `false`, or `disabled` so I had a 25% of getting it right. I'll change mine to keep it consistent

Reply:
Going to leave this as null for now. ESlint doesn't like false because it's a boolean and I hate to add another type just for that.